docs/build/wasp-evm/v1.0.0-rc.6/docs/how-tos/integrate-your-exchange.mdx

```diff
@@ -0,0 +1,162 @@
+
+### Interact with the ShimmerEVM Chain
+
+You can use [`wasp-cli`](/wasp-cli/how-tos/wasp-cli/) to safely interact with
+and [manage](wasp-cli/how-tos/chain-management/) your Wasp instance.
+
+## Creating Transactions
+
+ShimmerEVM handles transactions in the same way as standard EVM transaction with minor exceptions:
+
+- They must be signed with ShimmerEVM's ChainID (<ChainId url='https://json-rpc.evm.shimmer.network'/>).
+- The used EVM gas is converted to ISC gas before being charged to the sender.
+- EVM contracts can't receive on-ledger requests.
+
+:::warning Compatibility
+
+Please make sure that you review
+the [EVM Compatibility in IOTA Smart Contracts guide](../getting-started/compatibility.md#on-ledger-requests).
+
+:::
+
+### Determining Finality
+
+[Block times are variable in ShimmerEVM](../getting-started/compatibility.md#no-enforced-block-time). A new EVM "block"
+will be created only when an ISC block is created, and ISC does not enforce an average block time. This means that block
+times are variable; a new block will be created as soon as needed. The average block time is 2.8 seconds.
+
+## Tooling
+
+ShimmerEVM is [compatible with most wallet and tools](../getting-started/compatible-tools.md) used
+in EVM, such as [Metamask](../getting-started/compatible-tools.md#metamask), [Remix](../getting-started/compatible-tools.md#remix),
+[Hardhat](../getting-started/compatible-tools.md#hardhat) and [Ethers.js](../getting-started/compatible-tools.md#ethersjsweb3js).
+
+:::info ShimmerEVM Testnet
+
+The [ShimmerEVM Testnet](/build/networks-endpoints/#testnet-evm) is a great way to develop and test your integration
+without incurring in any risks or costs.
+
+:::
+
+## On-Chain Data
+
+ShimmerEVM support any standard way of consuming on-chain data you use for any other EVM network.
+
+## Oracle Support
+
+The ShimmerEVM network supports [Oracles](/build/oracles/) to retrieve data from outside the contract's sandbox, such as
+[Pyth](/build/oracles#pyth) and [Supra](/build/oracles#supra).
+
+## FAQs
+
+### What is the GitHub Address For the Network?
+
+SMR is the native gas token on ShimmerEVM and therefore has no GitHub repository. You can
+find [Hornet](https://github.com/iotaledger/hornet) and [Wasp](https://github.com/iotaledger/wasp) in
+the [IOTA Ledger GitHub](https://github.com/iotaledger/).
+
+### What Are the Hardware Requirements for Running a Node?
+
+You can find a detailed list of requirements in
+the [Hornet documentation](/hornet/getting_started/#recommended-requirements).
+
+### What's the Precision of Each Transaction (i.g 0.0001)?
+
+ShimmerEVM has the standard 18 decimals.
+
+### How Can I Customize RPC Port and the Directory of Block Data?
+
+The JSON RPC will be available by default on
+`<NODE_HOST>/wasp/api/v1/chains/smr1prxvwqvwf7nru5q5xvh5thwg54zsm2y4wfnk6yk56hj3exxkg92mx20wl3s/evm`
+
+You can route it however you wish using a proxy or any other stander tool for the task.
```

Comment:
this depends on whether they use their own hornet/wasp or not, right?...

btw, "stander tool"?